### PR TITLE
Updates for Site Security docs

### PR DESCRIPTION
**Pull request by jdrossl**
https://github.com/craftercms/craftercms/issues/3992

Please merge for 3.1 too

### CODE REVIEW

File: source/site-administrators/engine/engine-site-security-guide.rst [hunk 1]
```diff
@@ -117,322 +149,33 @@ be used:
...
-              var interval = setInterval(function() {
-                  if (fbDialog == null || fbDialog.closed) {
-                      clearInterval(interval);
-
-                      location.reload();
-                  }
-              }, 1000);
-          } catch(e) {}
-      }
+Once the authentication and authorization configurations are completed you can use the ``authToken`` object in
```
Comment (avasquez614) [inline source/site-administrators/engine/engine-site-security-guide.rst:156]: Isn't the base interface of `authToken` the Spring `Authentication` interface? Maybe we can mention that and link that Javadoc.

Comment (jdrossl) [reply source/site-administrators/engine/engine-site-security-guide.rst:156]: That's already documented in the freemarker/groovy variables, let me know if we should add a link to that or to the class directly

Comment (avasquez614) [reply source/site-administrators/engine/engine-site-security-guide.rst:156]: Let's do that to make it more clear

File: source/site-administrators/engine/engine-site-security-guide.rst [hunk 2]
```diff
@@ -117,322 +149,38 @@ be used:
...
+|
 
-      def additionalParams = new LinkedMultiValueMap<String, String>()
-          additionalParams.add("scope", "email,public_profile")
-          additionalParams.add("display", "popup")
+Migrating from Crafter Profile
+==============================
 
-      return "redirect:" + providerLoginSupport.start(siteContext.siteName, "facebook", request, additionalParams, connectSupport)
+Previous to version ``3.1.5`` Crafter Profile was the only security provider available, all sites created in previous
```
Comment (alhambrav) [inline source/site-administrators/engine/engine-site-security-guide.rst:176]: Maybe change `Previous` to `Prior`?

**Pull request merged**